2.4/Dockerfile

```diff
@@ -127,6 +127,7 @@ RUN set -eux; \
 		--build="$gnuArch" \
 		--prefix="$HTTPD_PREFIX" \
 		--enable-mods-shared=reallyall \
+		--enable-mpms-shared=all \
```

Comment:
[Alpine](https://github.com/docker-library/httpd/blob/49d553ae79f1b42ba541714c4e611aec5eefdfa8/2.4/alpine/Dockerfile#L110) needs this too. :wink:

Reply:
Updated!